notes.txt

```diff
@@ -1,6 +1,17 @@
 Needed this to run UnrealEd http://download.microsoft.com/download/vb50pro/utility/1/win98/EN-US/Msvbvm50.exe
 
 for v1.4========================
+I should probably abs when setting seed, Crc returns negative numbers
```

Comment:
I call abs before modulus, so it's probably fine, but I should make a test that checks that rng() >= 0